src/utils/HipaceProfilerWrapper.H

```diff
@@ -20,11 +20,14 @@
 #include <AMReX_BLProfiler.H>
 #include <AMReX_GpuDevice.H>
 
+/** Whether to call amrex::Gpu::synchronize() around all profiler region */
+inline int m_do_device_synchronize = 0;
```

Comment:
`m_` stands for "member variable". Should the prefix be removed here? Also, should we remove `m_do_device_synchronize` from class `Hipace`?